app/forms/otp_verification_form.rb

```diff
@@ -0,0 +1,22 @@
+class OtpVerificationForm
+  def initialize(user, code)
+    @user = user
+    @code = code
+  end
+
+  def submit
+    @success = user.authenticate_direct_otp(code)
```

Comment:
So, I moved the success to its own method, but that looks strange to me because now the only thing `submit` does is return `result`, but I think it reads better if `submit` processes the form submission inside the method, which is the pattern we've been using in other forms. Thoughts?

```ruby
class OtpVerificationForm
  def initialize(user, code)
    @user = user
    @code = code
  end

  def submit
    result
  end

  private

  attr_reader :code, :user

  def success?
    user.authenticate_direct_otp(code)
  end

  def result
    {
      success?: success?
    }
  end
end
```